### ISSUE TITLE
Add language support to /universe/names/

### ISSUE THREAD

**Issue by CarbonAlabel** [esi/esi-issues#996]
The /universe/names/ endpoint doesn't support the `Accept-Language` header or the `language` query parameter, so I can't use this endpoint if I want the names returned in a language other than English.

**Comment by Aidansavage** [comment]
Kinda hard to add language support for player, corporation, and alliance names when there isn't a way to translate them. For typeids the other endpoint does support the language arguments iirc.

**Comment by tsuthers-ccpgames** [comment]
Yeah, it should translate for systems/types/regions etc. For names there is no translation but that's fine.

Universe/ids alleges it supports the language parameter but that's actually just lies, both endpoints need to be fixed.

**Comment by evanova** [comment]
Any chance we can get this solved? For languages that (still) have translated items, if we only want to get the name or description, the only way to get them is looping through each item. This is not a viable solution since the SDE is also not including translations for Korean and has missing type names for German and Russian.
